k8s/helmfile/prometheus.yaml

```diff
@@ -0,0 +1,18 @@
+releases:
+  - name: kube-prometheus-stack
+    namespace: monitoring
```

Comment:
Putting this in a different namespace is stolen from #446 - do we want it?

Reply:
I think yes we do want this, we'd also want to put the prometheus exporters in this namespace as well as the Pod/Service monitoring defintions